packages/downloading-helpers/src/scripts/replays.ts

```diff
@@ -159,6 +159,32 @@ const downloadReplayV3Files = async (
     ];
   });
 
+  const diffsFolder = join(replayDir, "diffs");
+  await Promise.all(
+    Object.keys(diffs ?? {}).map((baseReplayId) =>
+      mkdir(join(diffsFolder, baseReplayId), { recursive: true })
+    )
+  );
+
+  const diffsPromises = Object.values(diffs ?? {}).flatMap((diffsForBase) => {
```

Comment:
The downloadScope === "screenshots" was initially added for [this line of code](https://github.com/alwaysmeticulous/meticulous/blob/847757ae40e6fb584cff3fd24c1b814ccc4ee284/packages/replay-orchestrator/src/replay/screenshot-diffing/download-and-diff-screenshots.ts#L189) so it'd only download the files needed when diffing screenshots. It looks like it isn't being used yet (didn't finish the swing on that one!), but might as well make this `const diffPromises = downloadScope === "everything" ? ... : []`, like line 188, since 'download-and-diff-screenshots' won't care about diff images.

Reply:
(we probably should update that call to pass the download scope, I suspect it'd make a reasonable perf difference)

Reply:
Opened a PR here https://github.com/alwaysmeticulous/meticulous/pull/2431

Reply:
Thanks, done! And approved your PR 👍🏼